feat(214537): Add separate SB3 / LogCloud example to work with 6.0

pascalleclercq opened the pull request.

pascalleclercq committed:
feat(214537): Add separate SB3 / LogCloud example to work with 6.0

pascalleclercq committed:
feat(214537): Add separate SB3 / LogCloud example to work with 6.0

configure latest release version of Atomikos Xtreme transaction 6

GuyPardon commented:
Do we really need a dependency on spring-boot-starter-parent? I would expect this to be pulled in by "our" starter?

pascalleclercq committed:
feat(214537): Add separate SB3 / LogCloud example to work with 6.0

configure latest release version of Atomikos Xtreme transaction 6
add missing lib transactions-hibernate4
